src/cdcacm_serial.c

```diff
@@ -171,7 +171,7 @@ void init_cdc_acm()
 	gpio_pin_write(gpio_dev, TXRX_LED, 1);
 }
 
-void cdcacm_setup(void)
+void cdcacm_setup(void *dummy1, void *dummy2, void *dummy3)
```

Comment:
@eriknyquist I actually figured out the reason for the warnings. I forgot to change the function prototypes to have three arguments also. I'll amend the request later today